### PR TITLE
Update products and product maps

### PR DESCRIPTION
**Pull request by redhatrises**

### CODE REVIEW

File: assets/src/app/ato/Products/Static.tsx [hunk 1]
```diff
@@ -2,9 +2,12 @@ export const ProductIdOverridesMap: {[Identifier: string]: string} = {
     coreos4: 'coreos-4',
     idm: 'identity-management',
     ocp3: 'openshift-container-platform-3',
+    ocp4: 'openshift-container-platform-4',
     osp13: 'openstack-platform-13',
+    rhel7: 'enterprise-linux-7',
+    rhel8: 'enterprise-linux-8',
```
Comment (isimluk) [inline assets/src/app/ato/Products/Static.tsx:8]: Why do you see a need adding those items?

This whole map is here only to maintain links that people may have bookmarked 10+ months ago. CommplianceAsCode/redhat project used to have different product names in both branches. After migration of ComplianceAsCode/redhat to github actions, I consolidated old names. This map is here only to keep the old names working. No need to add new ones.

File: assets/src/app/ato/Products/Static.tsx [hunk 2]
```diff
@@ -76,13 +79,13 @@ const ProductInfo: {[Identifier: string]: ProductTemplate } = {
             'SCAP': require('@app/assets/markdown/products/openstack-platform-13/SCAP.md').default,
         }
     },
-    'rhel-7': {
+    'enterprise-linux-7': {
```
Comment (isimluk) [inline assets/src/app/ato/Products/Static.tsx:82]: This change breaks the ui. Making `Overview` and `SCAP` sub-pages unavailable.

Not sure what we are trying to achieve here, but we need significant amount of changes to rename product to something else while keeping the ui working.

**Comment by isimluk** [comment]
I just noticed https://github.com/ComplianceAsCode/redhat/pull/1205

We may need to replace all the occurrences of `rhel-7` to enterprise-linux-7 in here. Similarly for `rhel-8`. Including directory names. Then we need to institute override from `rhel-7` to `enterprise-linux-7`.

Alternatively, we could just rename enterprise-linux-7 to rhel-7 in ComplianceAsCode, that would be significantly less labor intensive and thus preferable.

**Comment by redhatrises** [comment]
> I just noticed [ComplianceAsCode/redhat#1205](https://github.com/ComplianceAsCode/redhat/pull/1205)
> 
> We may need to replace all the occurrences of `rhel-7` to enterprise-linux-7 in here. Similarly for `rhel-8`. Including directory names. Then we need to institute override from `rhel-7` to `enterprise-linux-7`.
> 
> Alternatively, we could just rename enterprise-linux-7 to rhel-7 in ComplianceAsCode, that would be significantly less labor intensive and thus preferable.

The only thing is that we don't have consistent naming practices which makes it unfortunate.

**Pull request closed without merging**